ros_gz_sim/CMakeLists.txt

```diff
@@ -26,7 +27,6 @@ find_package(gz-msgs REQUIRED)
 find_package(gz_sim_vendor REQUIRED)
 find_package(gz-sim REQUIRED)
 # Needed in launch/gz_sim.launch.py.in
-set(GZ_SIM_VER ${gz-sim_VERSION_MAJOR})
```

Comment:
Can you rever this? This is still needed in https://github.com/gazebosim/ros_gz/blob/08e03897cae5d54b16bcbb618489d89ebed38625/ros_gz_sim/launch/gz_sim.launch.py.in#L78

Reply:
Restored in [01345e2](https://github.com/gazebosim/ros_gz/pull/528/commits/01345e2d95d76998f3ab8fbd341e2164d075a009).